packages/runtime/src/helpers/files.ts

```diff
@@ -340,6 +340,54 @@ const getServerFile = (root: string, includeBase = true) => {
   return findModuleFromBase({ candidates, paths: [root] })
 }
 
+/**
+ * Try to find next-server module in few locations (to support different next versions) and in few context (try to resolve from app location and from this module)
+ */
+export const getNextServerModulePath = (root: string): string | null => {
+  // first let's try to use app location directory to find next-server
+  try {
+    const nextServerModuleLocation = getServerFile(root, false)
+    if (nextServerModuleLocation) {
```

Comment:
For the guards we have here, would there ever be a case where `nextServerModuleLocation` would be falsy? Looking at the code, unless it throws because the module does not exist, we should always be get a value set in `nextServerModuleLocation` in all the try/catch if blocks here? Or maybe I'm missing a piece.

Reply:
`getServerFile` and more specifically `findModuleFromBase` that is called there

https://github.com/netlify/next-runtime/blob/44dac405ca887fa6b6b3e14f831c376a68d102ed/packages/runtime/src/helpers/utils.ts#L257-L272

doesn't throw - instead it returns falsy `null` if it can't find a module

Reply:
This was refactored (that function was actually removed) since initial commit so I'll resolve this, but general sentiment is still that a lot of those helper functions don't throw and instead return falsy `null` if module can't be resolved